src/code/z_vr_box.c

```diff
@@ -1,13 +1,677 @@
+            ASSERT(skyboxCtx->staticSegments[2] != NULL, "vr_box->vr_box_staticSegment[2] != NULL", "../z_vr_box.c",
+                   1566);
+
+            DmaMgr_SendRequest1(skyboxCtx->staticSegments[2], start, size, "../z_vr_box.c", 1567);
+            break;
+        case SKYBOX_HOUSE_SARIA:
+            skyboxCtx->unk_140 = 2;
+
+            start = _vr_K5VR_staticSegmentRomStart;
+            size = _vr_K5VR_staticSegmentRomEnd - start;
+            skyboxCtx->staticSegments[0] = GameState_Alloc(&globalCtx->state, size, "../z_vr_box.c", 1574);
+            ASSERT(skyboxCtx->staticSegments[0] != NULL, "vr_box->vr_box_staticSegment[0] != NULL", "../z_vr_box.c",
+                   1575);
+
+            DmaMgr_SendRequest1(skyboxCtx->staticSegments[0], start, size, "../z_vr_box.c", 1576);
+
+            start = _vr_K5VR_pal_staticSegmentRomStart;
+            size = _vr_K5VR_pal_staticSegmentRomEnd - start;
+            skyboxCtx->staticSegments[2] = GameState_Alloc(&globalCtx->state, size, "../z_vr_box.c", 1579);
+            ASSERT(skyboxCtx->staticSegments[2] != NULL, "vr_box->vr_box_staticSegment[2] != NULL", "../z_vr_box.c",
+                   1580);
+
+            DmaMgr_SendRequest1(skyboxCtx->staticSegments[2], start, size, "../z_vr_box.c", 1581);
+            break;
+        case SKYBOX_HOUSE_ALLEY:
+            skyboxCtx->unk_140 = 2;
+
+            start = _vr_KR3VR_staticSegmentRomStart;
+            size = _vr_KR3VR_staticSegmentRomEnd - start;
+            skyboxCtx->staticSegments[0] = GameState_Alloc(&globalCtx->state, size, "../z_vr_box.c", 1588);
+            ASSERT(skyboxCtx->staticSegments[0] != NULL, "vr_box->vr_box_staticSegment[0] != NULL", "../z_vr_box.c",
+                   1589);
+
+            DmaMgr_SendRequest1(skyboxCtx->staticSegments[0], start, size, "../z_vr_box.c", 1590);
+
+            start = _vr_KR3VR_pal_staticSegmentRomStart;
+            size = _vr_KR3VR_pal_staticSegmentRomEnd - start;
+            skyboxCtx->staticSegments[2] = GameState_Alloc(&globalCtx->state, size, "../z_vr_box.c", 1593);
+            ASSERT(skyboxCtx->staticSegments[2] != NULL, "vr_box->vr_box_staticSegment[2] != NULL", "../z_vr_box.c",
+                   1594);
+
+            DmaMgr_SendRequest1(skyboxCtx->staticSegments[2], start, size, "../z_vr_box.c", 1595);
+            break;
+        case SKYBOX_UNSET_27:
+            break;
+    }
+}
+#else
+void Skybox_Setup(GlobalContext* globalCtx, SkyboxContext* skyboxCtx, s16 skyboxId);
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_vr_box/Skybox_Setup.s")
+#endif
+
+void Skybox_Init(GlobalContext* globalCtx, SkyboxContext* skyboxCtx, s16 skyboxId) {
+    skyboxCtx->unk_140 = 0;
+    skyboxCtx->rot.x = 0.0f;
+    skyboxCtx->rot.y = 0.0f;
+    skyboxCtx->rot.z = 0.0f;
+
+    Skybox_Setup(globalCtx, skyboxCtx, skyboxId);
+    osSyncPrintf("\n\n\n＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊\n\n\nＴＹＰＥ＝%d"
+                 "\n\n\n＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊\n\n\n",
+                 skyboxId);
```

Comment:
Splitting the strings even further might be good here? Not sure
```suggestion
    osSyncPrintf("\n\n\n＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊\n\n\n"
                 "ＴＹＰＥ＝%d"
                 "\n\n\n＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊＊\n\n\n",
                 skyboxId);
```

Reply:
yeah I agree